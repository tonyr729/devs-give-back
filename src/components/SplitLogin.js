import React from 'react';

export const SplitLogin =() => {
  return (
    <div>
      <p> This is the split login page </p>
    </div>
  );
};
