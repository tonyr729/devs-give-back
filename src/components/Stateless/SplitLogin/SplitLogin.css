.client-login {
  color: #4f4f4f;
  background-color: #ffffff;
  height: 50vh; }
  .client-login a p {
    font-size: 5rem;
    padding: 12rem; }
    .client-login a p:hover {
      color: #29bbff; }

.dev-login {
  color: #f5f5f5;
  background-color: #03a9f4;
  height: 50vh; }
  .dev-login a p {
    font-size: 5rem;
    padding: 12rem; }
    .dev-login a p:hover {
      color: #ffffff; }
