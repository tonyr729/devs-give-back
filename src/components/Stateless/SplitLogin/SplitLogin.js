import React from 'react';
import { NavLink } from 'react-router-dom';
import './SplitLogin.css';

export const SplitLogin =() => {
  return (
    <div>
      <div className='client-login'>
        <NavLink to='/client-login'>
          <p>Problem Owner</p>
        </NavLink>
      </div>
      <div className='dev-login'>
        <NavLink to='/dev-login'>
          <p>Developer</p>
        </NavLink>
      </div>
    </div>
  );
};
