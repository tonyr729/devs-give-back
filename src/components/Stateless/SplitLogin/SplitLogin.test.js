import { shallow } from 'enzyme';
import React from 'react';
import { SplitLogin } from './SplitLogin'

describe('SplitLogin', () => {
  it('should match snapshot', () => {
    const splitLogin = shallow(<SplitLogin  />)

    expect(splitLogin).toMatchSnapshot();
  });
});