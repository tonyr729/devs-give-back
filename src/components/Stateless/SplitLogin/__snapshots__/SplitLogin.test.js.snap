// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SplitLogin should match snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Unknown />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <p>
         This is the split login page 
</p>,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": " This is the split login page ",
      },
      "ref": null,
      "rendered": " This is the split login page ",
      "type": "p",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <p>
           This is the split login page 
</p>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": " This is the split login page ",
        },
        "ref": null,
        "rendered": " This is the split login page ",
        "type": "p",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
